src/commands/dev/index.js

```diff
@@ -35,47 +34,59 @@ const startStaticServer = async ({ settings, log }) => {
   log(`\n${NETLIFYDEVLOG} Server listening to`, settings.frameworkPort)
 }
 
+const NOT_FOUND_EXIT_CODE = 127
+
+const isNonExistingCommandError = (error) => {
+  // `ENOENT` is only returned for non Windows systems
+  // See https://github.com/sindresorhus/execa/pull/447
+  // exitCode === 127 is returned from npm/yarn
+  if (error.code === 'ENOENT' || error.exitCode === NOT_FOUND_EXIT_CODE) {
+    return true
+  }
+
+  // this only works on English versions of Windows
+  return (
+    typeof error.message === 'string' && error.message.includes('is not recognized as an internal or external command')
+  )
+}
+
 const startFrameworkServer = async function ({ settings, log, exit }) {
   if (settings.noCmd) {
     return await startStaticServer({ settings, log })
   }
 
   log(`${NETLIFYDEVLOG} Starting Netlify Dev with ${settings.framework || 'custom config'}`)
-  const commandBin = await which(settings.command).catch((error) => {
-    if (error.code === 'ENOENT') {
-      throw new Error(
-        `"${settings.command}" could not be found in your PATH. Please make sure that "${settings.command}" is installed and available in your PATH`,
+
+  const commandWithArgs = `${settings.command} ${settings.args.join(' ')}`
```

Comment:
We might want to use `execa()` instead of `execa.command()`.
Note about `execa.command()`:
  - It does not run in a shell by default (like `execa()`)
  - If an argument has a significant space, it must be escaped with a backslash. In our case, I believe we would need to do this escaping on `settings.args`? Or we can just use `execa()` which does not have that problem.

Reply:
I used `execa.command` in preparation to use the output of `framework-info` which is a single string.
https://github.com/netlify/framework-info#example-nodejs

I'll switch to `execa` until we land https://github.com/netlify/cli/pull/1704

Reply:
Oh, then `execa.command()` makes sense. :+1: 
We might just want to escape spaces with a backslash if they might contain any. Something along the lines of:

```js
settings.args.map(arg => arg.replace(/ /g, '\\ '))
```

Reply:
I'm keeping `execa` for now in https://github.com/netlify/cli/pull/2857/commits/00eec5f8ef810d117e17fef2616e313ab7d785af

I think we might want to handle space escaping in `framework-info` so the command in the result is ready to be invoked